app/src/ereferrals/java/org/eyeseetea/malariacare/strategies/DashboardActivityStrategy.java

```diff
@@ -109,10 +110,8 @@ public DashboardActivityStrategy(DashboardActivity dashboardActivity) {
 
     @Override
     public void onCreate() {
-
         ICredentialsRepository iCredentialsRepository = new CredentialsLocalDataSource();
-
-        Credentials credentials = iCredentialsRepository.getLastValidCredentials();
+        credentials = iCredentialsRepository.getLastValidCredentials();
```

Comment:
@xurxodev please get the credentials using GetLastInsertedCredentialsUseCase instead of credentialsRepository

Reply:
@idelcano We will do this in the future because the OnCreate method is now complex executing severals use case and I think is more interesting review all OnCreate method I will create an issue to refactor this method whole

Reply:
@idelcano I will create an issue to refactor this because onCreate method is now very confuse executing several use cases